lib/rules/cli/4.1.yaml

```diff
@@ -147,6 +147,8 @@
     :f: -f <value>
     :filename: --filename=<value>
     :record: --record=<value>
+    :reource_type: <reource_type>
```

Comment:
it should be `resource_type` and `resource_name`

Reply:
@yapei thx, updated.

Reply:
There is no resouce_type ,when "oc create --help"